### ISSUE TITLE
How to pretrain/train the trRosetta network by myself?

### ISSUE THREAD

**Issue by SuperXiang** [Learn-Ventures/trdesign-pytorch#1]
Would you please also share the scripts of pretrain/train if I want to train the models with other datasets?

Thanks

**Comment by aiXander** [comment]
We did not train trRosetta ourselves, the original authors repo is here:
https://github.com/gjoni/trRosetta

Unfortunately, there is no code for training the model on a custom dataset so you will have to create those scripts yourself...
However, it is not super hard to do: essentially you need to:

- get a dataset of pdb's
- extract the needed 2D feature maps for each pdb: [dist, omega, phi, theta]
- Setup a model with the architecture from the code (use whatever backend you want)
- Read the original paper to pick the appriopriate loss metrics
- train the model
